### PR TITLE
Fixing quickmodel trace access

### PR DESCRIPTION
**Pull request by steven-albanese**
Looks like the problem is with the way the traces are being accessed. Opening a PR to see if I can fix this!

### CODE REVIEW

**Comment by steven-albanese** [comment]
Ok I've got this working locally now! We'll see if the tests pass, or if there's another issue.

**Comment by jchodera** [comment]
If you're updating to use a new aspect of the `pymc` API, make sure you pin to the minimum `pymc` version with this new API!

**Comment by jchodera** [comment]
(Unless, of course, this way of using the API is backwards-compatible!)

**Comment by steven-albanese** [comment]
Ok! I'll have to dig into the pymc2 docs and see when they switched over to this API. I'm not sure if it's backwards compatible or not

**Comment by steven-albanese** [comment]
Also -- where do I pin a package?

**Comment by jchodera** [comment]
Both the `build` and `run` sections [here](https://github.com/choderalab/assaytools/blob/master/devtools/conda-recipe/meta.yaml#L12-L48), and then later in the conda recipe!

**Comment by steven-albanese** [comment]
Ok -- I think I've correctly pinned the version of `pymc` we need, and I've also thrown in a quick script to plot traces from the `pickle` object that gets written out.

**Comment by jchodera** [comment]
@steven-albanese : Can you commit your changes to `devtools/conda-recipe/meta.yaml`?

File: devtools/conda-recipe/meta.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ requirements:
     - pandas
     - scipy
     - openblas ==0.2.19
-    - pymc
+    - pymc ==2.3.7
```
Comment (jchodera) [inline devtools/conda-recipe/meta.yaml]: Can you use `>=` instead of `==` here?

**Comment by steven-albanese** [comment]
I think this might be ok to merge unless there are other concerns!

**Comment by jchodera** [comment]
Nope!

**Pull request merged**